rails/Gemfile

```diff
@@ -15,3 +15,5 @@ gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails'
 
 gem 'pry'
+
+gem 'tzinfo-data', platforms: [:x64_mingw, :mingw, :mswin]
```

Comment:
I needed to add this as I couldn't get Rails server to work on Windows without out. Relevant stackoverflow question https://stackoverflow.com/questions/24674940/cannot-start-rails-server-due-to-tzinfo-gem